lib/videoroom/room.ex

```diff
@@ -133,9 +139,17 @@ defmodule Videoroom.Room do
 
     webrtc_extensions =
       if state.simulcast? do
-        [Mid, Rid, TWCC]
+        [
+          Mid,
+          Rid,
+          TWCC
+          # , VAD
+        ]
       else
-        [TWCC]
+        [
+          TWCC
+          # , VAD
+        ]
```

Comment:
Unfortunately, this looks bad, we need to handle this a bit better. Can you:
1. Add `enable_vad?: false` in `handle_init` to room's state
2. Conditionally append `VAD` to webrtc_extensions, i.e.
```elixir
webrtc_extensions =
  if state.enable_vad? do
    webrtc_extensions ++ [VAD]
  else
    webrtc_extensions
  end
```